test/cypress/integration/signup-and-login.spec.js

```diff
@@ -0,0 +1,65 @@
+describe('SignUp, Profile and Login', () => {
+  // NOTE: Create a uniqueId to avoid duplicated users on DB during crypress watch mode (during dev)
+  const userId = new Date().getMilliseconds()
+  const userName = `user1-${userId}`
+
+  it('successfully loads homepage', () => {
+    cy.visit('/')
+  })
+
+  // NOTE: What's the purpose of this test?
+  // it.skip('Should start the backend server if necessary', function () {
+  //   return require('../backend/index.js')
+  // })
+
+  it('sign up new user1', () => {
+    cy.giSignUp(userName)
```

Comment:
Missing commented out test:

```
it.skip('Should fail to create George again'
```

All commented out tests need to not be lost

Reply:
If you noticed, I added that test in the last place at the end of this file:

```
  it.skip('user1 must not signup twice', () => {})
```

and I did the same for the other tests as well. To give you some piece, you have my word that I'll do all the tests missing during the next weeks ;)